beamer/agent/config.py

```diff
@@ -110,9 +113,13 @@ def load(config_path: Path, options: dict[str, Any]) -> Config:
         raise ConfigError(f"missing settings: {missing}")
 
     rpc_urls = {}
+    chain_confirmation_blocks = {}
 
     for chain_name, chain_info in config["chains"].items():
         rpc_urls[chain_name] = URL(chain_info["rpc-url"])
+        chain_confirmation_blocks[chain_name] = int(
+            chain_info.get("confirmation-blocks", config["confirmation-blocks"])
+        )
```

Comment:
I'm guessing this `int(...)` here is only because of mypy... if so, :woozy_face: